packages/manager/modules/pci/src/components/order-command/index.js

```diff
@@ -15,7 +15,7 @@ angular
     'ngTranslateAsyncLoader',
     'pascalprecht.translate',
   ])
-  .component('databaseOrderCommand', component)
+  .component('orderCommand', component)
```

Comment:
**issue:** Component name looks too generic. Is there any reason to rename it here?

Reply:
Yes as it won't be linked to databases anymore. The idea is to make this component available to everyone in public cloud. Would PCIOrderCommand work better for you ?

Reply:
Yes looks good!

Reply:
Any update @Dimfacion

Reply:
Yeah, sorry took me a bit long to find some time for this minor change. Should be good now :)